src/app/myworkflows/myworkflows.service.ts

```diff
@@ -198,13 +198,16 @@ export class MyWorkflowsService extends MyEntriesService<Workflow, OrgWorkflowOb
   /**
    * Grabs the workflow/service/notebook from the webservice and loads it
    * @param workflow Selected workflow
+   * @param entryType
    */
   selectEntry(workflow: DockstoreTool | Workflow | null, entryType: EntryType | null): void {
     if (workflow && entryType && workflow.id) {
-      this.workflowsService.getWorkflow(workflow.id, includesValidation + ',' + includesAuthors).subscribe((result: Workflow) => {
-        this.location.go('/my-' + entryType + 's/' + result.full_workflow_path);
-        this.workflowService.setWorkflow(result);
-      });
+      this.workflowsService
+        .getWorkflow(workflow.id, [includesValidation, includesAuthors, includesMetrics].join(','))
+        .subscribe((result: Workflow) => {
+          this.location.go('/my-' + entryType + 's/' + result.full_workflow_path);
+          this.workflowService.setWorkflow(result);
+        });
```

Comment:
PR looks good, Charles!
Orthogonal grumble about the code reformatter generating this big diff, which made us have to visually compare each of the lines one-by-one to determine that the only change was that metrics were added to the request.

Reply:
Yeah, good point, I should have paid closer attention to that; I dislike PRs that have formatting changes that hide the actual change.

Reply:
> Yeah, good point, I should have paid closer attention to that; I dislike PRs that have formatting changes that hide the actual change.

Not your fault at all, Charles!  Pretty sure the way the reformatter works, there's no way to avoid it, it joins the lines until they pass its length maximum, and then it breaks them apart, resulting in the big diffs.  I personally don't care for it, it's turned some pretty readable code of mine into a jumble.

Reply:
I often look at the diff in GitHub before assigning reviewers, and try to fix things like that if I feel the formatting is "inappropriate", but didn't do it this time.